lighthouse-cli/compiled-check.js

```diff
@@ -6,7 +6,7 @@ const path = require('path');
 module.exports = function(filename) {
   if (!fs.existsSync(path.join(__dirname, filename))) {
     console.log('Oops! Looks like the CLI needs to be compiled. Please run:');
-    console.log('   npm run install-cli');
+    console.log('   npm run build-cli');
```

Comment:
if `install-cli` has `build-cli` in it now, should this stay `install-cli` then?